documentation/staging/content/userguide/base-images/patch-images.md

```diff
@@ -127,20 +125,11 @@ The `rebase` command does the following:
 Using `rebase`, the new image can be created in one of two ways:
 
 - As a new WebLogic image from a base OS image (similar to the `create` command; recommended).
+  **NOTE**:  Oracle strongly recommends rebasing your images with the latest security patches by applying
+  the [`--recommendedPatches`](https://oracle.github.io/weblogic-image-tool/userguide/tools/rebase-image/) option.
+
+- Or, as a base image, use WebLogic Server CPU images from OCR that do not already have a domain home.
 
-  To activate:
-  - Set `--tag` to the name of the final new image.
-  - Set `--sourceImage` to the WebLogic image that contains the WebLogic configuration.
-  - Set additional fields (such as the WebLogic and JDK locations),
-    similar to those used by `create`.
-    See [Create a custom base image](#create-a-custom-base-image).
-  - Do _not_ set `--targetImage`.  (When
-    you don't specify a `--targetImage`, `rebase` will use
-    the same options and defaults as `create`.)
```

Comment:
Not sure why this "To activate" was removed. This first three bullets removed may seem obvious, but they help at least a little IMO - especially for new users, the fourth one caused some customer/user confusion, which is why it's there.

Reply:
@tbarnes-us We'll need to confer on this one. I'll ping you on Slack.